src/pages/home/home.ts

```diff
@@ -16,10 +17,56 @@ export class HomePage {
   segment: Actions = this.actions.rent;
   tag: string = '';
 
-  constructor(public navCtrl: NavController, public stockpileData: StockpileData) { }
+  constructor(
+    public navCtrl: NavController,
+    public alertCtrl: AlertController,
+    public stockpileData: StockpileData,
+    public inventoryData: InventoryData
+  ) { }
 
   onNext() {
     if (this.tag) {
+      if (this.segment === this.actions.add) {
+        this.pushPage('');
```

Comment:
This allows to add items. Otherwise, the call for status below returns a 404 since the tag does not exist yet in the db and it does not push the page to add an item on the nav stack. It also needs to be passing an empty string because of the comparisons in the method (`status.toLowerCase()` would crash with `toLowerCase()` is not a function).

However, this is all temporary, because add and edit items will be moved to the inventory page with #104 in the next sprint.